integration-tests/testconfig/common/vrf/common.go

```diff
@@ -0,0 +1,297 @@
+package vrf
+
+import (
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/pkg/errors"
+
+	"github.com/smartcontractkit/chainlink-testing-framework/blockchain"
+)
+
+type Config struct {
+	Common            *Common            `toml:"Common"`
+	General           *General           `toml:"General"`
+	ExistingEnvConfig *ExistingEnvConfig `toml:"ExistingEnv"`
+	Performance       *PerformanceConfig `toml:"Performance"`
+}
+
+const (
+	ErrDeviationShouldBeLessThanOriginal = "`RandomnessRequestCountPerRequestDeviation` should be less than `RandomnessRequestCountPerRequest`"
+)
+
+func (c *Config) Validate() error {
+	if c.Common != nil {
+		if err := c.Common.Validate(); err != nil {
+			return err
+		}
+	}
+	if c.General != nil {
+		if err := c.General.Validate(); err != nil {
+			return err
+		}
+	}
+	if c.Performance != nil {
+		if err := c.Performance.Validate(); err != nil {
+			return err
+		}
+	}
+
+	return nil
+}
+
+type Common struct {
+	CancelSubsAfterTestRun *bool `toml:"cancel_subs_after_test_run"`
+}
+
+func (c *Common) Validate() error {
+	return nil
+}
+
+type PerformanceConfig struct {
+	TestDuration          *blockchain.StrDuration `toml:"test_duration"`
+	RPS                   *int64                  `toml:"rps"`
+	RateLimitUnitDuration *blockchain.StrDuration `toml:"rate_limit_unit_duration"`
+}
+
+func (c *PerformanceConfig) Validate() error {
+	if c.TestDuration == nil || c.TestDuration.Duration == 0 {
+		return errors.New("test_duration must be set to a positive value")
+	}
+	if c.RPS == nil || *c.RPS == 0 {
+		return errors.New("rps must be set to a positive value")
+	}
+	if c.RateLimitUnitDuration == nil {
```

Comment:
is a duration of `0` a valid one? I think it is, right?

Reply:
I dont remember, need to ask @skudasov